Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -76,6 +76,10 @@ public double translateTypicalShootingPositionToDistance(TypicalShootingPosition
         }
     }
     
+    public double getTargetSpeedForRange(double rangeInInches) {
+        return flushToBoilerTargetSpeed.get() + trimFlushToBoilerSpeed.get();
```

Comment:
I think it's mostly aspirational - eventually it'd be nice to dynamically shoot for a few different ranges (possibly coming from the distance sensor) but right now we just have one range